.env.template

```diff
@@ -0,0 +1,21 @@
+####
+# ETL Settings
+#
+# If these environment variables are set it will enable the ETL portion of the automated setup.
+# It will download data and various OTS utilities to support that process.
+export OTS_SVN_USERNAME=""
```

Comment:
I believe it is.  To be clear, are you saying that Bourne is the least common denominator and should be used?

FWIW In other projects where I've seen an `.env.template` (such as [this one](https://github.com/PermanentOrg/devenv/blob/main/.env.template)) it appears the template assumes bash.  Happy to switch to the Bourne approach.

Especially because it means I get to write more lines of code too, which means I'll get a really great performance review!

Reply:
Unhnh.  I mean, we probably have this problem in a lot of other places, but let's not make it worse.  Yes, let's assume Bourne but not Bash.  There are plenty of systems where /bin/sh is not Bash.